addons/service/multimedia/streamdisc-server/source/bin/streamdisc-server.start

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+################################################################################
+#      This file is part of OpenELEC - http://www.openelec.tv
+#      Copyright (C) 2009-2014 Stephan Raue (stephan@openelec.tv)
+#
+#  OpenELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  OpenELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with OpenELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+. /etc/profile
+
+oe_setup_addon service.multimedia.streamdisc-server
+
+ADDON_SETTINGS="$ADDON_HOME/settings.xml"
+
+chmod a+x $ADDON_DIR/bin/*
```

Comment:
Not needed, see https://github.com/OpenELEC/OpenELEC.tv/blob/master/packages/mediacenter/kodi/profile.d/99-kodi.conf